portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,7 +29,20 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Data page</h1>");
-    response.getWriter().println("<p>Hello Giovanni</p>");
+    // response.getWriter().println("<h1>Data page</h1>");
+    // response.getWriter().println("<p>Hello Giovanni</p>");
+
+    ArrayList<String> s = new ArrayList<String>(3);
```

Comment:
```suggestion
    List<String> s = new ArrayList<String>(3);
```

Couldn't find this in the style guide, but generally it's preferable to use the interface type rather than the specific type. (This makes it easier to change to a different `List` type later if needed)